explore/src/main/scala/explore/attachments/ObsAttachmentsTable.scala

```diff
@@ -115,19 +115,17 @@ object ObsAttachmentsTable extends TableHooks with ObsAttachmentUtils:
                                  oa.description,
                                  dom.readReadableStream(IO(f.stream()))
             ) *>
-            props.obsAttachments
-              .mod(
-                _.updatedWith(oa.id)(
-                  _.map(
-                    _.copy(fileName = name,
-                           updatedAt = Timestamp.unsafeFromInstantTruncated(Instant.now()),
-                           checked = false
+            IO(Timestamp.unsafeFromInstantTruncated(Instant.now())).flatMap { now =>
```

Comment:
We have this
https://github.com/gemini-hlsw/explore/blob/master/workers/src/main/scala/workers/package.scala#L46

Though it may need to be moved to be reused

Reply:
I moved `IO.now()` into the model project (explore.model.syntax.all). It could be used by the workers, but doesn't seem to be at this point.

Reply:
👍